index.html

```diff
@@ -1,8 +1,153 @@
+         <li><a href = #nugget>Nugget</a> | </li>
+         <li><a href = #accio>Accio</a> | </li>
+         <li><a href = #kahlua>Kahlua</a> | </li>
+         <li><a href = #pebbles>Pebbles</a></li>
+         </nav></ul>
+
+      <h4>Dogs</h4>
+         <nav><ul>
+         <li><a href = #bud>Bud</a> | </li>
+         <li><a href = #malibu>Malibu</a></li>
+         </ul></nav>
+         </header>
+
+      
+      <main>
+         
+      <h1>Cats</h1>
+         
+      <article id = "callieholly">
+      <h2>Callie & Holly</h2>
+      <p>Callie came to my family as a stray when she was about a year old. She was the first animal I really have core memories with. She was an avid mouser and loved exploring the outdoors. Callie remained a well-fed outdoor cat until, one day, she seemingly dropped all of her weight. I was five at the time and didn't really notice, but my parents did. A few weeks later, while doing yardwork, my mother noticed a tiny black and white face peering at her from a small hole in the door of a shed on our property. Behold, Callie had given birth to six kittens of varying colors and patterns. Out of this litter, we kept the runt, the lone other calico cat. I named her Holly because I loved the name and had heard my sister talk about one of her friends named Holly. (Unfortunatly, human Holly would later pass away. I befriended her daughter as adults and shared with her the story of how I inadvertently named a cat after her late mom.) Both cats have since crossed the rainbow bridge but they started my love for cats in general but especially calicos.</p>
+      
+      
+      <figure>
+         <img src="images\CALLIE.jpg" width="400" height = "300">
+         <figcaption>Callie, enjoying time in the outdoors.</figcaption>
+      </figure>
+
+      <figure>
+        <img src="images\HOLLY.jpg">
+         <figcaption>Holly, napping on a table.</figcaption>
+      </figure>
+      </article>
+      
+      <center><br><a href = #top>Back to Top</a></center>
+      
+
+      <article id = "nugget">
+      <h2>Nugget</h2>
+      <p>In the winter of 2013, I was working at a retreat center in the Ozarks when the camp cat gave birth to yet another litter of kittens. This was a litter of three, all black jellybeans. Unfortunately, due to an extreme cold snap, only one kitten survived and her mother rejected her. I adopted her as my office mascot and named her Nugget. When I left the camp in 2014, Nugget came with me. Despite having been an indoor/outdoor cat for the first year and a half of her life, Nugget was more than content to become a fat, lazy house cat. Due to the trauma of her early kittenhood, Nugget is not easily trusting and has some anger issues. We've tried working them out but it usually ends in bloodshed. Approaching ten years old, Nugget is still content with being an only cat and lives with my parents where she enjoys yelling at them for food, snuggling with my dad, and tormenting my mom.</p>
```

Comment:
same as comment above ^ hitting enter every few sentences or so will give the code some more readability :)